packages/client/mod.ts

```diff
@@ -52,14 +52,21 @@ export type Patch =
   }
   & (
     | {
-      Op: "add" | "remove" | "replace" | "append";
+      Op: "add" | "remove" | "replace" | "append" | "increment" | "decrement";
```

Comment:
a bit unrelated but inc/dec to take a value (the amount you want to change, so you can do += 5 with this for example)